backend/pong/game_logic.py

```diff
@@ -0,0 +1,113 @@
+# pong/game_logic.py
+from .constants import *
+import math
+
+
+class PongGame:
+    def __init__(self):
+        # Initialize game state
+        self.player1_y = PLAYGROUND_HEIGHT // 2  # Initial position for player 1 (midpoint)
```

Comment:
Im thinking as it is pong, maybe player_left and player_right would be more descriptive than player1 and player2...

Reply:
When wanting to rename anything like functions or variables in VS Code you can right click a highlighted symbol and select 'Rename symbol' and VS Code will find all references to that symbol and rename them:
![image](https://github.com/dorianjagusch/ft_transcendence/assets/44494010/3f419095-0d10-4dec-9a6a-1220b6a486ef)